src/stream/src/executor/over_window/frame_finder.rs

```diff
@@ -143,6 +147,73 @@ pub(super) fn find_frame_end_for_rows_frame<'cache>(
     find_boundary_for_rows_frame::<false /* RIGHT */>(frame_bounds, part_with_delta, curr_key)
 }
 
+/// Given the first and last key in delta, calculate the order values of the first
+/// and the last frames logically affected by some `RANGE` frames.
+pub(super) fn calc_logical_curr_for_range_frames(
```

Comment:
Managed to understand the function, but didn't quite get the point of "logical"

Reply:
By "logical (current key)" I mean the current key will be affected if it exists, but in actual we may not find the logical current key in the cache + delta, hence consider the closest one as the "actual current key".